README.md

```diff
@@ -89,6 +89,8 @@ Possible values:
 
 - `eu` (default): For spaces created in the EU
 - `us`: For spaces created in the US
+- `ca`: For spaces created in Canada
```

Comment:
Same comment as in https://github.com/storyblok/storyblok-js/pull/413#pullrequestreview-1779575459